src/styles/base/_fonts.scss

```
@@ -0,0 +1,48 @@
+// Font faces
+
+// Circular Std font faces
+// book - (font-weight: 450 , font-style: book)
+// bold - (font-weight: 700 , font-style: bold)
+// black - (font-weight: 900 , font-style: black)
+
+@font-face {
+    font-family: 'Circular Std';
+    src:
+        url(/assets/fonts/circular-std/CircularStd-Book.ttf) format('truetype'),
+        url(/assets/fonts/circular-std/CircularStd-Book.eot) format('eot'),
+        url(/assets/fonts/circular-std/CircularStd-Book.woff) format('woff'),
+        url(/assets/fonts/circular-std/CircularStd-Book.woff2) format('woff2');
+    font-weight: 450;
```

```suggestion
    font-weight: 400;
```

use 400 weight for regular font

---

We need 450, 700 and 900 as per figma and 450 was named with book but coderabbit gives error for that